build/uberjar/build.clj

```diff
@@ -0,0 +1,12 @@
+(ns uberjar.build
+  (:require [hf.depstar.uberjar])
+  (:import (javafx.application Platform)))
+
+;; see https://github.com/cljfx/cljfx#aot-compilation-is-complicated
+
+(defn run
+  [& args]
+  (try
+    (apply hf.depstar.uberjar/run args)
+    (finally
+      (Platform/exit))))
```

Comment:
Can we avoid this file altogether by using `-Dcljfx.skip-javafx-initialization=true` JVM option instead?

Reply:
`-Dcljfx.skip-javafx-initialization=true` wasn't sufficient... the compile hung w/ a Javafx platform icon in my dock. But using `Platform/exit` here does work. (What do you think? I know you'd explicitly avoided AOT, so I can take the AOT out - though it is probably something of general interest..)

Reply:
An idea to think about... which would spare this common pattern and having to have `build/` dir... But cljfx could offer a `cljfx.build/with-platform-exit` function that this `(try (apply ... (finally (Platform/exit))` pattern and respected `:exec-fn` and `:exec-args`...

So then here we (and cljfx + deps + aot users) could simply do something like: 

```
 :aliases {:uberjar {:extra-deps {seancorfield/depstar {:mvn/version "1.1.136"}}
   :replace-paths ["src" "build"]
   :exec-fn cljfx.build/with-platform-exit
   :exec-args {:exec-fn hf.depstar.uberjar/run
                       :exec-args {:jar "dist/hn.jar"
                                           :aot true
                                           :main-class "hn.core"}}}}
```

Then we wouldn't need the custom `/build` dir and script.